adodbapi/apibase.py

```diff
@@ -564,7 +564,7 @@ def __next__(self):
             yield self._getValue(n)
 
     def __repr__(self):  # create a human readable representation
-        taglist = sorted(list(self.rows.columnNames.items()), key=lambda x: x[1])
+        taglist = sorted(self.rows.columnNames.items(), key=lambda x: x[1])

```

Comment:
```suggestion
        taglist = sorted(self.rows.columnNames.values())
```
Same change as in ##2332